swift/schema.py

```diff
@@ -228,7 +228,8 @@ class ParamDecl(VarDecl):
     """)
 
 class Callable(Element):
-    name: optional[string] | doc("name of this callable")
+    name: optional[string] | doc("name of this callable") | desc("The name includes any arguments "
+        "of the callable, for example `myFunction(arg:)`.")
```

Comment:
To stick with the right Swift terminology we should probably be more precise what we mean by "argument" here. Swift has both argument labels, and parameter names (see [here](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions#Function-Argument-Labels-and-Parameter-Names)), and I _think_ the name we produce for a function declaration uses the argument label.

Reply:
Yes, it's the argument labels here.  Parameter names are internal implementation details of the function and nobody else should care about them.

I've reworded slightly (and fixed the merge conflict).